src/app/page.module.css

```diff
@@ -1,9 +1,61 @@
-.main {
+.mainpageContainer {
+  background: #1b1228;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 6rem;
-  min-height: 100vh;
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  padding: 0;
+}
+
+.welcomecontainer {
+  width: 353px;
+  height: 92px;
+  margin-top: 1px;
+  maring-left: 19px;
+  gap: 0px;
+  opacity: 0px;
+  font-family: Cascadia Mono;
```

Comment:
delete, no need to define width/height as well as margins of a h1 element. Also font family is already defined in globals for all text so there isn't a need to redefine it here too.